modules/sandboxed-containers-create-kata-config-resource-cli.adoc

```diff
@@ -28,26 +28,26 @@ Kata is installed on all worker nodes by default. If you want to install `kata`
 
 . Create a YAML file with the following manifest:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
 apiVersion: kataconfiguration.openshift.io/v1
 kind: KataConfig
 metadata:
   name: cluster-kataconfig
 spec:
-  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
+  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:{sandboxed-containers-version}.0
```

Comment:
Should the two references in this module be `sandboxed-containers-version-z`?

Reply:
I originally thought so, but @jensfr told me these images are not updating with the z-stream version. They remain 1.x.0 even when there are z-stream updates.

Reply:
Same with the must-gather image.